Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -531,6 +531,9 @@ public static void RegisterRoutes(IKernel kernel, RouteCollection routes)
             routes.MapHttpRoute("get-masterkey", "api/functions/admin/masterkey", new { controller = "Function", action = "GetMasterKey" }, new { verb = new HttpMethodConstraint("GET") });
             routes.MapHttpRoute("delete-function", "api/functions/{name}", new { controller = "Function", action = "Delete" }, new { verb = new HttpMethodConstraint("DELETE") });
 
+            // Docker Hook Endpoint
+            routes.MapHttpRoute("docker", "docker/hook", new { controller = "Docker", action = "ReceiveHook" }, new { verb = new HttpMethodConstraint("POST") });
```

Comment:
That's what I'm saying - can you completely skip the route registration here if we're on Windows?